srcs/config/ConfigServer.cc

```diff
@@ -4,11 +4,13 @@
 #include <limits>
 
 ConfigServer::ConfigServer()
-    : port_(),
-      client_max_body_size_(std::numeric_limits<size_t>::max()),
-      routes_(),
+    : routes_(),
       error_pages_(),
-      files_() {}
+      files_{},
+      server_name_{"localhost"},
```

Comment:
We can't initialize server_name_ to localhost, an empty server_name_ is allowed and we'll run into problems with servers using the same host:port combo (check out ListenSocket::addConfig)